### DISCUSSION TITLE
Welcome to manifesto Discussions!

### DISCUSSION THREAD

**Discussion by SebastianStadil** [opentffoundation/manifesto#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MrKamranA** [comment]
Hello there! I want to express my gratitude for the hard work and commitment that has gone into the OpenTF Foundation. Thank you for all your efforts! 

I have a question that may seem obvious - does the team behind the OpenTF Foundation believe that HashiCorp will take note of the substantial support for the initiative and perhaps reconsider their decision to change to the BSL license? Additionally, if HashiCorp does revert the change, do you think the community might lose faith in their ability to keep their products open source in the future after this sudden change, and will the OpenTF foundation continue operating because of this?
